cumulus_library/study_parser.py

```diff
@@ -503,8 +515,37 @@ def run_single_table_builder(
         if not name.endswith(".py"):
             name = f"{name}.py"
         self._load_and_execute_builder(
-            name, cursor, schema, verbose, drop_table=True, parser=parser
+            name, cursor, schema, verbose=verbose, drop_table=True, parser=parser
+        )
+
+    def run_generate_sql(
+        self,
+        cursor: databases.DatabaseCursor,
+        schema: str,
+        parser: databases.DatabaseParser = None,
+        **kwargs,
+    ) -> None:
+        """Loads modules from a manifest and executes code via BaseTableBuilder
+
+        :param cursor: A DatabaseCursor object
+        :param schema: The name of the schema to write tables to
+        :param verbose: toggle from progress bar to query output
+        """
+        all_generators = (
+            self.get_table_builder_file_list()
+            + self.get_counts_builder_file_list()
+            + self.get_statistics_file_list()
         )
```

Comment:
This feels like something that could get stale. Should there be a utility method that grabs all these and used in multiple places? Or does that just not make sense for this setup.

Reply:
This is currently the only place that needs to know about all the table builder types in one go. If there was another place, I'd agree. I'm inclined to wait until that happens?